<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAChartAddtGridRasterDataWidget</class>
 <widget class="QWidget" name="DAChartAddtGridRasterDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add XY Series</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBoxX">
         <property name="title">
          <string>X</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxX"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBox">
         <property name="title">
          <string>Y</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxY"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBoxMatrics">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Matrics</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxMatrics"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableViewRaster"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DADataManagerComboBox</class>
   <extends>QComboBox</extends>
   <header>DADataManagerComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>DA::DACollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">DACollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
